### ISSUE TITLE
[SOLVED] Hotfix 10 / Patch 4 makes BGSE start in a loop

### ISSUE THREAD

**Issue by MaloDrex** [Norbyte/bg3se#155]
Hello,

Since the hotfix 10 was published today, the BSGE starts in a loop (the console window opens as usually, but the game does not starts, and the console window reopen again and again).

Thanks in advance for your help.

Best regards.

**Comment by MitoZ** [comment]
> Hello,
> 
> Since the hotfix 10 was published today, the BSGE starts in a loop (the console window opens as usually, but the game does not starts, and the console window reopen again and again).
> 
> Thanks in advance for your help.
> 
> Best regards.

it's because of patch 4, this mod is not supported yet, just turn this mod off and all other mods based on this

**Comment by MaloDrex** [comment]
I will quietly wait for its update.
Thank you for your answer :-)


Le jeu. 9 nov. 2023 à 21:42, Dzmitry Barkouski ***@***.***> a
écrit :

**Comment by SheeraDarkphoenix** [comment]
that's what I am doing. I am waiting for the BG3SE update so I can play. Till then, I will be playing other games. Keep up the hard work, Norbyte. Your hard work is appreciated.

**Comment by Norbyte** [comment]
Hi,
I've checked and this does not seem reproducible to me, game always launches properly

**Comment by MaloDrex** [comment]
Here is what i see (since this morning, the BG3SE window does not starts in
loop, it juste run once, but the game  does not launch):

[image: 2023-11-10_12h24_13.png]

Best regards.

**Comment by kweeky** [comment]
> Hi, I've checked and this does not seem reproducible to me, game always launches properly

Game also loads for me successfully  (with only two mods: TutorialChestSummoning and UniversalArmourFixer):

```BG3Ext v9 built on Nov  5 2023 00:50:24
Game version v4.39.11.62 OK```

@MaloDrex are you seeing both of these in your Script Extender Debug Console window or are the values different? (your attached image didn't work)

**Comment by MaloDrex** [comment]
Here is my screenshot, on imgur:

https://imgur.com/a/ubwLQ7K

Malodrex.

**Comment by Eudoxxi** [comment]
I am having this same issue my save gets to 60 - 80% then just kicks me to the home page. This is what the command window shows 

![Screenshot (435)](https://github.com/Norbyte/bg3se/assets/149214026/4756d645-196b-4d16-ac9a-0740aa30fe4a)

**Comment by kweeky** [comment]
> I am having this same issue my save gets to 60 - 80% then just kicks me to the home page. This is what the command window shows
> 
> ![Screenshot (435)](https://user-images.githubusercontent.com/149214026/282086695-4756d645-196b-4d16-ac9a-0740aa30fe4a.png)

Not the same issue if you're being kicked out back to the main menu when loading a save (the issue reported here is BG3 won't even start at all for @MaloDrex)

With your issue, you have several mods enabled, have you tried removing those SE mods and loading your save with just BG3SE? It could be an issue with one of those if they haven't been updated for the latest patch.

**Comment by Eudoxxi** [comment]
> > I am having this same issue my save gets to 60 - 80% then just kicks me to the home page. This is what the command window shows
> > ![Screenshot (435)](https://user-images.githubusercontent.com/149214026/282086695-4756d645-196b-4d16-ac9a-0740aa30fe4a.png)
> 
> Not the same issue if you're being kicked out back to the main menu when loading a save (the issue reported here is BG3 won't even start at all for @MaloDrex)
> 
> With your issue, you have several mods enabled, have you tried removing those SE mods and loading your save with just BG3SE? It could be an issue with one of those if they haven't been updated for the latest patch.

my bad i read his post wrong.  i have tried what you said issues still happens, will just wait for update on this.

**Comment by Yetropolos** [comment]
I have the same issue as MaloDrex with the same loadout in his screenshot provided, after hotfix 10, it just starts tries to load the game, console closes itself then the launcher does the broken mods spiel.

Ive tried launching with each mod stripped out as well as only with BG3SE and then just with vanilla after verifying constantly.

Currently seeing if a full reinstall will shake some branches.

**Comment by MaloDrex** [comment]
UPDATE a new build id was pushed on steam version after Hotfix 10: 12674212.
Now, the game crashes at start (a bug report window triggers, but it says no crash log will be attached to the report).
I can run any recent game on my PC, except BG3, so, that's not my hardware.
I did a clean reinstall of the game (uninstall, removing any remaining steam subfolder and larian appdata folder too), i asked steam to check file integrity, nothing fixes it. I filed a bug report to Larian.

**Comment by Pyr0blad3** [comment]
select v10 in BG3 MM -> Settings -> Preferences -> Updater -> Update Channel: Devel -> Select Version: Latest
maybe it helps you.

**Comment by MaloDrex** [comment]
I do not have an "Updater" submenu in BG3MM (v1.0.10.0), latest available
using the update function of BG3MM itself.



Le dim. 12 nov. 2023 à 17:03, NightShyft ***@***.***> a
écrit :

**Comment by Yetropolos** [comment]
@Pyr0blad3  managed to get a tad further in my troubleshooting thanks to your suggestion, now im just at a spot where BG3MM doesn't recognize 5 of my 6 mods.

But hey, plus side it doesn't merry go round trying to launch a couple of times opening and closing before crashing anymore xD

**Comment by MaloDrex** [comment]
[SOLVED] Since the latest nvidia driver (546.17) is installed the game launches.
I have no longer problems with mods requiring BG3SE (i used the download and extract BG3SE command from BG3 Mod Manager to install BG3SE).
